src/controllers/gym.js

```diff
@@ -90,6 +90,8 @@ class Gym extends Controller {
 			data.googleMapUrl = `https://www.google.com/maps/search/?api=1&query=${data.latitude},${data.longitude}`
 			data.appleMapUrl = `https://maps.apple.com/maps?daddr=${data.latitude},${data.longitude}`
 			data.wazeMapUrl = `https://www.waze.com/ul?ll=${data.latitude},${data.longitude}&navigate=yes&zoom=17`
+			data.reactMapUrl = `${this.config.general.reactMapURL}${!this.config.general.reactMapURL.endsWith('/') ? '/' : ''}id/gyms/${data.gym_id}`
```

Comment:
Whilst your code is fine, there is a general problem with people who don't update default.json (usually docker users).  In this case it would fail.

you could just do `if (this.config.general.reactMapURL) data.reactMapUrl...` which would protect the code from a missing entry (endswidth on undefined would throw an exception)